src/utilities/color.ts

```diff
@@ -29,4 +33,29 @@ export function hexStringToRgba(colorString: string, alpha: number = 1): RGBA {
         a: alpha
     };
 }
-// end stolen from https://stackoverflow.com/questions/21646738/convert-hex-to-rgba
\ No newline at end of file
+// end stolen from https://stackoverflow.com/questions/21646738/convert-hex-to-rgba
+
+// return color map as Uint8ClampedArray according colorMap
+function initContextWithSize(width: number, height: number) {
+    const canvas = document.createElement("canvas") as HTMLCanvasElement;
+    canvas.width = width; 
+    canvas.height = height; 
+    const ctx = canvas.getContext("2d");
+    return ctx;
+}
+
+export function getColorsForValues (colorMap: string): {color: Uint8ClampedArray, size: number} {
+    const colorMaps = RenderConfigStore.COLOR_MAPS_ALL;
+    const colorMapIndex = colorMaps.indexOf(colorMap);
+
+    // !!!colorMap image size (1024,790)
```

Comment:
this should be re-worded. Perhaps:
`// the source image for colormaps is 1024x790, with each colormap taking a 1024x10 region`

Reply:
Thanks, fixed now.